tools/extract_build_defines.py

```diff
@@ -160,7 +160,7 @@ def ValidateAttrs(config_name, config_attrs, file_path, linenum):
                             # allow commas to appear inside brackets by converting them to and from NULL chars
                             _attrs = re.sub(r'(\(.+\))', lambda m: m.group(1).replace(',', '\0'), _attrs)
 
-                            if '=' in config_description:
+                            if '=' in config_description and not '==' in config_description:
```

Comment:
Can we revert these changes please? The parsing here is already pretty fragile, and I don't want to risk making it more fragile.
```suggestion
                            if '=' in config_description:
```